cli/internal/terraform/terraform/aws/variables.tf

```diff
@@ -0,0 +1,68 @@
+variable "name" {
+  type        = string
+  description = "Name of your Constellation"
+}
+
+variable "iam_instance_profile_worker_nodes" {
+  type        = string
+  description = "Name of the IAM instance profile for worker nodes"
+}
+
+variable "iam_instance_profile_control_plane" {
+  type        = string
+  description = "Name of the IAM instance profile for control plane nodes"
+}
+
+variable "instance_type" {
+  type        = string
+  description = "Instance type for worker nodes"
+  default     = "t3.micro"
+}
+
+variable "state_disk_type" {
+  type        = string
+  description = "EBS disk type for the state disk of the nodes"
+  default     = "gp2"
+}
+
+variable "state_disk_size" {
+  type        = number
+  description = "Disk size for the state disk of the nodes [GB]"
+  default     = 30
+}
+
+variable "control_plane_count" {
+  type        = number
+  description = "Number of control plane nodes"
+  default     = 1
+}
+
+variable "worker_count" {
+  type        = number
+  description = "Number of worker nodes"
+  default     = 1
+}
+
+variable "ami" {
+  type        = string
+  description = "AMI ID"
+  default     = "ami-02f3416038bdb17fb" // Ubuntu 22.04 LTS
```

Comment:
Should we remove all defaults which are required by the CLI anyway (node numbers, ami, region, instance-type, ...) Or should we keep those if e.g. we want to use the standalone terraform in the future?

Reply:
imo most variables should not have default values in the terraform templates, especially ones like ami, since those would need to be changed with every release

Reply:
Removed.